static/bidder-info/screencore.yaml

```diff
@@ -0,0 +1,14 @@
+endpoint: 'http://h1.screencore.io/?kp={{.AccountID}}&kn={{.SourceId}}'
+maintainer:
+  email: 'connect@screencore.io'
```

Comment:
@screencore-io 

prebid team has sent verification mail on connect@screencore.io . Please respond back with "received" message to complete verification